k8s/client.go

```diff
@@ -1084,6 +1085,22 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string) (string, error) {
+	client := action.NewGetValues(c.HelmActionConfig)
```

Comment:
Rather than `client`, this is more accurately an `action`.

Reply:
https://github.com/helm/helm/blob/8854547d3518cf3c83eec80d9390a05c453f4b3e/cmd/helm/get_values.go#L42

This is called client in helm...

Reply:
it returns a `GetValues` struct, there are different structs in action pkg, and I will rename it to `helmGetValsClient` hope its more accurate.

Reply:
Since there is an “action config”, it would makes sense to call this an “action” (does it satisfy an Action interface maybe?). Then it makes more sense for an action to “Run” rather than for a client to “Run”, usuall a client would “request” or “get” in this case.

Either way, if the official codebase is doing it, this is fine : )

Reply:
I already updated it to `helmGetValsClient` in my latest commit and pushed my commit last week. 

btw, this is not something urgent, and I can wait for next week for more discussion or review